DllImportGenerator/DllImportGenerator/Marshalling/BoolMarshaller.cs

```diff
@@ -57,17 +55,14 @@ public IEnumerable<StatementSyntax> Generate(TypePositionInfo info, StubCodeCont
             switch (context.CurrentStage)
             {
                 case StubCodeContext.Stage.Setup:
-                    if (info.IsManagedReturnPosition)
-                        nativeIdentifier = context.GenerateReturnNativeIdentifier();
-
                     yield return LocalDeclarationStatement(
                         VariableDeclaration(
                             AsNativeType(info),
                             SingletonSeparatedList(VariableDeclarator(nativeIdentifier))));
 
                     break;
                 case StubCodeContext.Stage.Marshal:
-                    // <nativeIdentifier> = (<nativeType>)(<managedIdentifier> ? 1 : 0);
+                    // <nativeIdentifier> = (<nativeType>)(<managedIdentifier> ? _trueValue : _falseValue);
```

Comment:
Thank you! I really appreciate these small code snippets that distill the Roslyn code. There should really be one for every complex statement using the Roslyn API.